LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -189,25 +195,41 @@ namespace litecore {
 
     SQLiteDataFile::~SQLiteDataFile() { close(); }
 
+    // A lot of this logic can be turned into a reusable function later if needed
+    // for more extensions.
     static void LoadVectorSearchExtension(sqlite3* sqlite) {
 #ifdef COUCHBASE_ENTERPRISE
+        static const char* extensionName = "CouchbaseLiteVectorSearch";
```

Comment:
maybe constexpr

Reply:
This will make no difference since there is no evaluation that can be done on this at compile time.  It's memory address is used, which collapses it into a `static const char *` anyway,